storage_helper.cpp

```diff
@@ -22,19 +22,17 @@
 #define TRACE_GROUP "SMCS"
 
 StorageHelper::StorageHelper(BlockDevice *bd, FileSystem *fs)
-    : _bd(bd), _fs(fs)
+    : _bd(bd), _fs(fs), fs1(NULL), fs2(NULL), part1(NULL), part2(NULL)
 {
```

Comment:
What's the usage of the extra parameters?

Reply:
those are not new, need then in case we are using partition mode
but was not initialized